### DISCUSSION TITLE
Version 1.28 loop reboot on Synology DSM 7.2

### DISCUSSION THREAD

**Discussion by LolHa-FR** [dani-garcia/vaultwarden#3539] [General]
I installed DSM version 7.2 yesterday. The container manager has changed with this version.
I took the opportunity to upgrade from 1.27 to latest.
The container reboots in a loop.
I tested with 1.28 too and it's the same.
I had to revert to 1.27 for everything to work.

**Comment by BlackDex** [comment]
Please provide some logs, we can't do anything without it.

**Comment by Bloopps** [comment]
I am using version 1.28.1 and I have just migrated to DSM 7.2, all works correctly with this update

**Comment by CamelT0E** [comment]
No problem with the version on the Diskstation.
Tested on different devices with DSM 7.1.1 and DSM 7.2. It also worked with the DSM 7.2 beta and RC!

The fault must be somewhere else with you!
